### PR TITLE
SPLICE-1512 Multiple distinct aggregates

### PR DESCRIPTION
**Pull request by yxia92**
Please refer to comment SPLICE-1512 for explanation of the implementation.

### CODE REVIEW

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/GroupByNode.java [hunk 1]
```diff
@@ -627,10 +627,10 @@ private void addAggregates() throws StandardException{
...
         int numDistinct=numDistinctAggregates(aggregateVector);
         if(numDistinct!=0){
-            if(SanityManager.DEBUG){
-                SanityManager.ASSERT(groupingList!=null || numDistinct==1,
-                        "Should not have more than 1 distinct aggregate per Group By node");
-            }
+ //           if(SanityManager.DEBUG){
+ //               SanityManager.ASSERT(groupingList!=null || numDistinct==1,
+ //                       "Should not have more than 1 distinct aggregate per Group By node");
+ //           }
```
Comment (jleach4) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/GroupByNode.java]: Remove

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java [hunk 2]
```diff
@@ -618,9 +618,9 @@ public void bindExpressions(FromList fromListParam) throws StandardException{
 		** in the select list.
 		*/
         numDistinctAggs=numDistinctAggregates(selectAggregates);
-        if(numDistinctAggs>1){
-            throw StandardException.newException(SQLState.LANG_USER_AGGREGATE_MULTIPLE_DISTINCTS);
-        }
+//        if(numDistinctAggs>1){
```
Comment (jleach4) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java]: Remove

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/GroupByNode.java (hunk 1 above)
Comment (yxia92) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/GroupByNode.java]: Removed.

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java (hunk 2 above)
Comment (yxia92) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SelectNode.java]: Removed.

**Pull request merged**